browser/js/signup/signup.html

```diff
@@ -8,20 +8,38 @@
 
     <md-input-container>
         <label for="email">Email address</label>
-        <input type="text" ng-model="signup.email" class="form-control" id="email" placeholder="enter email" required>
+        <input type="text" ng-model="signup.email" required>
     </md-input-container>
     <md-input-container>
         <label for="login-password">Password</label>
-        <input type="password" ng-model="signup.password" class="form-control" id="login-password" placeholder="enter password" required>
+        <input type="password" ng-model="signup.password" required>
     </md-input-container>
     <md-input-container>
         <label for="first-name">First name</label>
-        <input type="text" ng-model="signup.firstName" class="form-control" id="first-name" placeholder="enter first name" required>
+        <input type="text" ng-model="signup.firstName" required>
     </md-input-container>
     <md-input-container>
         <label for="last-name">Last name</label>
-        <input type="text" ng-model="signup.lastName" class="form-control" id="last-name" placeholder="enter last name" required>
+        <input type="text" ng-model="signup.lastName" required>
     </md-input-container>
+    <div>Billing Address</div>
```

Comment:
We might want to consider making the address not be a required field in the AuthUser data model. That way they don't have to enter the billing address on sign-up. In this case, address could just be a required aspect of the checkout form.